cli/connect/console.go

```diff
@@ -247,6 +247,14 @@ func getExecutor(console *Console) prompt.Executor {
 		var data string
 		if _, err := console.conn.Eval(consoleEvalFuncBody, args, opts); err != nil {
 			if err == io.EOF {
+				if strings.Contains(console.input, "os.exit(") {
```

Comment:
Should we check the reason of EOF? Or it is okay to run the command below always? `os.exit` can be invoked using alias, or pcall, which will not be handled correctly.

Reply:
Rewrote. I think it should be fine to not check the reason of EOF.